Eatery/Views/Eateries/HistogramViewController.swift

```diff
@@ -0,0 +1,285 @@
+//
+//  HistogramViewController.swift
+//  Eatery
+//
+//  Created by Ethan Fine on 4/26/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+// TODO: Create a protocol that encapsulates all Swipe Data and make HistogramViewController independent of Eatery variables
+class HistogramViewController: UIViewController {
+    
+    // MARK: Data
+    var swipeData: EaterySwipeData!
+    // hours in overflowable military time units
+    let hourRange = (startTime: 6, endTime: 3)
+    var overflowingEndTime: Int!
+    
+    // MARK: Views and view properties
+    var viewFrame: CGRect!
+    var axisView: UIView!
```

Comment:
if these view's don't get initialized multiple times, we should make it into something like `private let axisView = UIView()`